packages/common/src/system/system-utils.ts

```diff
@@ -38,4 +41,34 @@ export namespace System {
             .toString('hex')
             .substr(0, length);
     }
+
```

Comment:
move this to a separate class. GuidUtils or something.

Reply:
refactored